apps/qtmozembed/declarativewebpage.cpp

```diff
@@ -273,16 +266,7 @@ void DeclarativeWebPage::setVirtualKeyboardMargin(qreal margin)
     qCDebug(lcCoreLog) << "WebPage: setting vkb margins:" << margin;
     if (margin != m_virtualKeyboardMargin) {
         m_virtualKeyboardMargin = margin;
-        if (m_virtualKeyboardMargin == 0.0) {
-            // Only place where we ignore view margins update guards.
-            // It must be allowed to close vkb while content is moving.
-            resetViewMargins();
-        } else {
-            QMargins margins;
-            margins.setBottom(m_virtualKeyboardMargin);
-            qCDebug(lcCoreLog) << "WebPage: set vkb margins:" << m_virtualKeyboardMargin;
-            setMargins(margins);
-        }
+        updateViewMargins();
```

Comment:
As far as I can tell, this has no effect without #741. If that is the case, I wonder if at this stage it might better to combined the two PRs, to avoid adding code (which has no effect) and then immediately removing it again.

There may be a good reason for it, but looking at this PR in isolation I'm finding it hard to figure out.